### PR TITLE
#174797453 fix internationalization for multi-languages

### PR DESCRIPTION
**Pull request by dushimeemma**
#### What does this PR do?
fix multi-languages support
#### Description of Task to be completed?
- move the locales file to the root directory of the project
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend/`
- enter into the directory `cd script-heroes-bn-backend`
- run build `npm run build`
- run the app `npm run start`
- send request on `{{url}}` using `postman`
- set headers `Accept-language:kin`
#### What are the relevant pivotal tracker stories?
[#174797453](https://www.pivotaltracker.com/n/projects/2465784/stories/174797453)
#### Screenshots (if appropriate)
Screenshot from 2020-10-08 08-21-42

### CODE REVIEW

**Review by ericus123 — APPROVED** [review]
Good

**Review by Habinezajanvier — APPROVED** [review]
Awesome

File: src/database/migrations/20200928155141-create-access-token.js [hunk 1]
```diff
@@ -1,26 +0,0 @@
-module.exports = {
-  up: async (queryInterface, Sequelize) => {
```
Comment (kipropbrian) [inline src/database/migrations/20200928155141-create-access-token.js:2]: Why are you deleting this file?

Comment (dushimeemma) [reply src/database/migrations/20200928155141-create-access-token.js:2]: It is no longer used, I was created it for storing the token on login but I was wrong with tables naming conversion and we created another migration for that.

**Pull request merged**